src/modules/createTodo.js

```diff
@@ -1,5 +1,38 @@
 import updateLocalStorage from './checkbox.js';
 
+function changeCompleted(input) {
+  const tasklist = input.parentElement.children;
+  const array = JSON.parse(localStorage.getItem('to-do'));
+
+  for (let i = 0; i < tasklist.length; i += 1) {
+    if (tasklist[i] === input) {
+      if (array[i].completed === false) {
+        array[i].completed = true;
+      } else {
+        array[i].completed = false;
+      }
+    }
+  }
+  localStorage.setItem('to-do', JSON.stringify(array));
+  /* eslint-disable-next-line */
```

Comment:
- [ ] **Optional:** Linters are meant to enforce community standards in Microverse and your future workplaces. Kindly fix the errors pointed out by the linters.